code/espurna/sensors/BME680Sensor.h

```diff
@@ -0,0 +1,287 @@
+          BSEC_OUTPUT_RAW_TEMPERATURE,                      // Raw temperature (ºC).
+
+          BSEC_OUTPUT_RAW_PRESSURE,                         // Raw pressure (Pa).
+
+          BSEC_OUTPUT_RAW_HUMIDITY,                         // Raw relative humidity (%).
+
+          BSEC_OUTPUT_RAW_GAS,                              // Gas resistance (Ohm). The resistance value changes according to the
+                                                            // VOC concentration (the higher the concentration of reducing VOCs,
+                                                            // the lower the resistance and vice versa).
+
+          BSEC_OUTPUT_IAQ,                                  // Indoor Air Quality, scaled based on the recent sensor history, ideal
+                                                            // for mobile applications (e.g. carry-on devices). The scale ranges from
+                                                            // 0 (clean air) to 500 (heavily polluted air). The automatic background
+                                                            // calibration process ensures that consistent IAQ performance is achieved
+                                                            // after certain of days (depending on BSEC configuration - 4d or 28d).
+
+          BSEC_OUTPUT_STATIC_IAQ,                           // Unscaled Indoor Air Quality, optimized for stationary applications
+                                                            // (e.g. fixed indoor devices).
+          BSEC_OUTPUT_CO2_EQUIVALENT,                       // Estimate of CO2 measured in the air.
+          BSEC_OUTPUT_BREATH_VOC_EQUIVALENT,                // Breath VOC represents the most important compounds in an exhaled
+                                                            // breath of healthy humans.
+
+          BSEC_OUTPUT_SENSOR_HEAT_COMPENSATED_TEMPERATURE,  // Temperature compensated for the influence of sensor heater (ºC).
+
+          BSEC_OUTPUT_SENSOR_HEAT_COMPENSATED_HUMIDITY,     // Relative humidity compensated for the influence of sensor heater (%).
+
+          BSEC_OUTPUT_STABILIZATION_STATUS,                 // Indicates initial stabilization status of the gas sensor element:
+                                                            // ongoing (0) or finished (1).
+
+          BSEC_OUTPUT_RUN_IN_STATUS,                        // Indicates power-on stabilization status of the gas sensor element:
+                                                            // ongoing (0) or finished (1).
+        };
+        double _breathVocEquivalent = 0.0f;
+        double _co2Equivalent = 0.0f;
+        double _gasResistance = 0.0f;
+        double _humidity = 0.0f;
+        double _iaq = 0.0f;
+        double _pressure = 0.0f;
+        double _rawHumidity = 0.0f;
+        double _rawTemperature = 0.0f;
+        double _temperature = 0.0f;
+        uint _iaqAccuracy = 0;
+        uint _iaqStatic = 0;
```

Comment:
```suggestion
        uint8_t _iaqAccuracy = 0;
        uint8_t _iaqStatic = 0;
```
?

Reply:
iaqStatic can go to 500 so I had a local change to `uint16_t`

Reply:
ref. src/bsec.h, it is float